recipes/medcoupling/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -ex
+
+mkdir -p build
+cd build
+
+if [[ "$mpi" == "nompi" ]]; then
+  on_mpi="OFF"
+else
+  on_mpi="ON"
+fi
+
+# remove the share cmake files
+find ${PREFIX}/share/cmake -type d -name "medfile-*" -exec rm -rf {} +
+
+cmake .. \
+    -DCMAKE_BUILD_TYPE="Release" \
+    -DPYTHON_ROOT_DIR="${PREFIX}" \
+    -DPYTHON_EXECUTABLE:FILEPATH="$PYTHON" \
+    -Wno-dev \
+    -DCONFIGURATION_ROOT_DIR="${SRC_DIR}/deps/config" \
+    -DSALOME_CMAKE_DEBUG=ON \
+    -DMED_INT_IS_LONG=ON \
```

Comment:
This line is in conflict with:

https://github.com/conda-forge/libmed-feedstock/blob/main/recipe/bld.bat#L7C14-L7C14

Reply:
Yeah, I think we could add this to libmed cmake build `-DMED_MEDINT_TYPE=long `

Reply:
> actually, libmed cmake file has that line, i'm going to try to cleanup a few things in [conda-forge/libmed-feedstock#57](https://github.com/conda-forge/libmed-feedstock/pull/57)

Thanks, I really appreciate your help!